src/game.cpp

```diff
@@ -6898,13 +6908,15 @@ void game::zones_manager()
         wrefresh( w_zones_info );
 
         tripoint center = u.pos() + u.view_offset;
-        const cata::optional<tripoint> first = look_around( w_zones_info, center, center, false, true );
+        cata::optional<peek_action_opt> peek_action = cata::nullopt;
```

Comment:
You don't need to explicitly initialize an `optional`, the default constructor already creates an empty object.

Reply:
This got removed entirely as part of refactoring per your other feedback.